packages/core/src/toolkit-provider/ToolkitProvider.tsx

```diff
@@ -25,31 +23,13 @@ export interface ToolkitContextProps {
 
 const DEFAULT_THEME_NAME = "light";
 
-declare global {
-  // eslint-disable-next-line @typescript-eslint/no-namespace
-  namespace JSX {
-    interface IntrinsicElements {
-      "uitk-theme": DetailedHTMLProps<
-        DOMAttributes<HTMLDivElement> & { class: string },
-        HTMLDivElement
-      >;
-    }
-  }
-}
-
 export const DensityContext = createContext<Density>(DEFAULT_DENSITY);
 
 export const ThemeContext = createContext<Theme[]>([]);
 
 export const BreakpointContext =
   createContext<Breakpoints>(DEFAULT_BREAKPOINTS);
 
-export const ToolkitContext = createContext<ToolkitContextProps>({
-  density: undefined,
-  themes: [],
-  breakpoints: DEFAULT_BREAKPOINTS,
-});
-
```

Comment:
Why are we removing this?

Reply:
That was an accidental leftover from splitting the ToolkitContext into separate contexts for Density, Breakpoints and theme ( see PR #602  )